drivers/flash/CMakeLists.txt

```diff
@@ -12,37 +12,14 @@ zephyr_library_sources_ifdef(CONFIG_SOC_FLASH_SAM flash_sam.c)
 zephyr_library_sources_ifdef(CONFIG_SOC_FLASH_NIOS2_QSPI soc_flash_nios2_qspi.c)
```

Comment:
CMake is complaining on Line 1 here, btw because of the empty library when building for the NRFXLIB LL.

Reply:
Fixed